src/main/twirl/uk/gov/hmrc/play/views/helpers/dateInputText.scala.html

```diff
@@ -0,0 +1,44 @@
+@*
```

Comment:
don't think we need this file, a normal @input should be used, like in the existing date helpers